sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_v1/query.sql

```diff
@@ -0,0 +1,123 @@
+WITH client_first_seen AS (
+  SELECT
+    client_id,
+    sample_id,
+    normalized_channel,
+    app_build,
+    app_channel,
+    app_display_version,
+    device_manufacturer,
+    device_model,
+    isp,
+    locale,
+    city,
+    country,
+    MIN(first_seen_date) AS first_seen_date
+  FROM
+    mozdata.fenix.baseline_clients_first_seen
+  WHERE
+    submission_date = DATE_SUB(@submission_date, INTERVAL 6 DAY)
+  GROUP BY
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    10,
+    11,
+    12
+),
+client_search AS (
```

Comment:
Just to make sure I understand this one, we're grabbing the number of searches by client_id that have occurred over the past 4 days, as long as the `first_seen_date` was on or before the start of that period?

Do we even need to do that second check (`DATE_DIFF(submission_date, first_seen_date, DAY) BETWEEN 4 and 7`)? You pulled from the `clients_first_seen` table of 6 days ago, so I think all values of `first_seen_date` will be at least 6 days ago?

Reply:
Yes, I should be able to remove the second check